esmvaltool/recipes/cmorizers/recipe_era5.yml

```diff
@@ -0,0 +1,343 @@
+      operator: mean
+  daily_min:
+    daily_statistics:
+      operator: min
+  daily_max:
+    daily_statistics:
+      operator: max
+
+diagnostics:
+  # hourly:
+  #   description: CMORize hourly ERA5 data
+  #   scripts:
+  #     script:
+  #       script: cmorizers/era5.py
+  #   variables:
+  #     clt:
+  #       mip: E1hr
+  #       era5_name: total_cloud_cover
+  #       era5_freq: hourly
+  #     evspsbl:
+  #       mip: E1hr
+  #       era5_name: evaporation
+  #       era5_freq: hourly
+  #     evspsblpot:
+  #       mip: E1hr
+  #       era5_name: potential_evaporation
+  #       era5_freq: hourly
+  #     mrro:
+  #       mip: E1hr
+  #       era5_name: runoff
+  #       era5_freq: hourly
+  #     orog:
+  #       mip: fx
+  #       era5_name: orography
+  #       era5_freq: hourly
+  #     pr:
+  #       mip: E1hr
+  #       era5_name: total_precipitation
+  #       era5_freq: hourly
+  #     prsn:
+  #       mip: E1hr
+  #       era5_name: snowfall
+  #       era5_freq: hourly
+  #     psl:
+  #       mip: E1hr
+  #       era5_name: mean_sea_level_pressure
+  #       era5_freq: hourly
+  #     ps:
+  #       mip: E1hr
+  #       era5_name: surface_pressure
+  #       era5_freq: hourly
+  #     ptype:
+  #       mip: E1hr
+  #       era5_name: precipitation_type
+  #       era5_freq: hourly
+  #     rlds:
+  #       mip: E1hr
+  #       era5_name: surface_thermal_radiation_downwards
+  #       era5_freq: hourly
+  #     rls:
+  #       mip: E1hr
+  #       era5_name: mean_surface_net_long_wave_radiation_flux
+  #       era5_freq: hourly
+  #     rsds:
+  #       mip: E1hr
+  #       era5_name: surface_solar_radiation_downwards
+  #       era5_freq: hourly
+  #     rsdt:
+  #       mip: E1hr
+  #       era5_name: toa_incident_solar_radiation
+  #       era5_freq: hourly
+  #     rss:
+  #       mip: E1hr
+  #       era5_name: surface_net_solar_radiation
+  #       era5_freq: hourly
+  #     tas:
+  #       mip: E1hr
+  #       era5_name: 2m_temperature
+  #       era5_freq: hourly
+  #     tasmax:
+  #       mip: E1hr
+  #       era5_name: maximum_2m_temperature_since_previous_post_processing
+  #       era5_freq: hourly
+  #     tasmin:
+  #       mip: E1hr
+  #       era5_name: minimum_2m_temperature_since_previous_post_processing
+  #       era5_freq: hourly
+  #     tdps:
+  #       mip: E1hr
+  #       era5_name: 2m_dewpoint_temperature
+  #       era5_freq: hourly
+  #     ts:
+  #       mip: E1hr
+  #       era5_name: skin_temperature
+  #       era5_freq: hourly
+  #     tsn:
+  #       mip: E1hr
+  #       era5_name: temperature_of_snow_layer
+  #       era5_freq: hourly
+  #     uas:
+  #       mip: E1hr
+  #       era5_name: 10m_u_component_of_wind
+  #       era5_freq: hourly
+  #     vas:
+  #       mip: E1hr
+  #       era5_name: 10m_v_component_of_wind
+  #       era5_freq: hourly
+  daily:
+    description: Create daily ERA5 data
+    scripts:
+      script:
+        script: cmorizers/era5.py
```

Comment:
I'm confused: if this is supposed to cmorize data on the fly, why you need a cmorizer here?
Isn't this supposed to work with any diagnostic script using the ERA5 data?

Reply:
Yes, that could work. This recipe is simply there for people that do like to keep a copy of their cmorized data on disk (for example, because it takes a long time). For one of our own use cases, it is very useful to keep the daily aggregations on disk.